src/azure-cli/azure/cli/command_modules/appconfig/_validators.py

```diff
@@ -30,11 +30,22 @@
 
 
 def validate_datetime(namespace):
-    ''' valid datetime format:YYYY-MM-DDThh:mm:ssZ '''
-    datetime_format = '^[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])T(2[0-3]|[01][0-9]):[0-5][0-9]:[0-5][0-9][a-zA-Z]{0,5}$'
-    if namespace.datetime is not None and re.search(datetime_format, namespace.datetime) is None:
-        raise CLIError(
-            'The input datetime is invalid. Correct format should be YYYY-MM-DDThh:mm:ssZ ')
+    ''' valid datetime format: YYYY-MM-DDThh:mm:ss["Z"/±hh:mm]'''
+    supported_formats = ["%Y-%m-%dT%H:%M:%S", "%Y-%m-%dT%H:%M:%Sz", "%Y-%m-%dT%H:%M:%S%z"]
+    if namespace.datetime is not None:
+        for supported_format in supported_formats:
+            if __tryparse_datetime(namespace.datetime, supported_format):
+                return
+
+        raise CLIError('The input datetime is invalid. Correct format should be YYYY-MM-DDThh:mm:ss["Z"/±hh:mm].')
```

Comment:
```suggestion
        raise InvalidArgumentValueError('The input datetime is invalid. Correct format should be YYYY-MM-DDThh:mm:ss["Z"/±hh:mm].')
```
Please use the specific error type `InvalidArgumentValueError` instead of `CLIError`

Reply:
Updated, thanks!